app/components/StationDetailView.vue

```diff
@@ -242,6 +244,33 @@ export default Vue.extend({
                 },
             });
         },
+        generateNotificationsFromPortalErrors() {
+            const portalError = this.currentStation?.portalHttpError;
+
+            if (this.$s.state.portal.currentUser && portalError?.name) {
+                const userId = this.$s.state.portal.currentUser.portalId;
+                const stationId = this.currentStation.id;
+
+                this.$s.dispatch(ActionTypes.ADD_NOTIFICATION, {
```

Comment:
I've started to use strongly typed actions more and more and I think they really help keep things stable or from getting brittle. No need to change this, but I'm trying not to add anymore that aren't. You can see them all in actions.ts, at the bottom of the file. It helps keep the payload/arguments matched up on the dispatching and handling side.